docs/components/FSXABaseAppLayout.md

```diff
@@ -0,0 +1,49 @@
+[Readme](../../README.md) » [BaseComponents](../README.md) » FSXABaseAppLayout
+
+# FSXABaseAppLayout
+
+1. [Introduction](#introduction)
+2. [Usage](#usage)
+3. [Properties](#properties)
+4. [Methods](#methods)
+
+## Introduction
+
+The FSXABaseAppLayout is there to have a basic component to be able to implement the general layout of the app.
+Thereby different attributes are given, which should simplify the development significantly. These are described in this chapter.<br />
```

Comment:
This sentence starting with "Thereby" should probably be rewritten (and correspondingly in all FSXABase... documentation files). Maybe start with "It provides different attributes, ..."?